### PR TITLE
Widget web analytics docs

### PR DESCRIPTION
**Pull request by filipkrw**
See: https://github.com/algomo/platform/pull/156

### CODE REVIEW

File: docs/docs/widgets/Web Analytics/Overview.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+| Contact details submitted (Algomo Widget) | -                                                | When a user submits their contact details in the Algomo widget. This event might get triggered multiple times by a single user, i.e. when they update their details |
+| Link clicked (Algomo Widget)              | `href`, `label`                                  | When a user clicks a link in the Algomo widget, either in a message or message sources.                                                                             |
+
+All events also have the following parameters: `widgetId`, `userName`, `userEmail`, `userPhone`, `userCompanyName`, `userBrowser`, `userMainLanguage`, `userLanguages`, `userPlatform`, `userTimeZone`. Some of them might be empty.
+
+### Custom events
+
+Currently, there's no way to track custom events. If you need to track custom events, please contact us. You can use our widget in the bottom right corner.
```
Comment (paul-algomo) [inline docs/docs/widgets/Web Analytics/Overview.md:22]: I'd go with:

---
Tracking custom events is not supported currently. If you'd like to track custom events, please get in touch via [email](mailto:support@algomo.com) or chat with us directly through the widget in the bottom corner.

Comment (paul-algomo) [reply docs/docs/widgets/Web Analytics/Overview.md:22]: Or, just don't mention custom events at all

File: docs/docs/widgets/Web Analytics/Overview.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+| Conversation started (Algomo Widget)      | `conversationId`, `conversationLanguage`         | When a user starts a conversation in the Algomo widget.                                                                                                             |
+| Contact details submitted (Algomo Widget) | -                                                | When a user submits their contact details in the Algomo widget. This event might get triggered multiple times by a single user, i.e. when they update their details |
+| Link clicked (Algomo Widget)              | `href`, `label`                                  | When a user clicks a link in the Algomo widget, either in a message or message sources.                                                                             |
+
+All events also have the following parameters: `widgetId`, `userName`, `userEmail`, `userPhone`, `userCompanyName`, `userBrowser`, `userMainLanguage`, `userLanguages`, `userPlatform`, `userTimeZone`. Some of them might be empty.
```
Comment (paul-algomo) [inline docs/docs/widgets/Web Analytics/Overview.md:18]: I'd be tempted to include a table of all parameters explaining what they do. Not a deal breaker though, most of them are self-explanatory.

File: docs/docs/widgets/Web Analytics/Google Tag Manager.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+3. Enter the parameter name under "Data Layer Variable Name", i.e. "widgetId".
+4. Save the variable.
+
+You can now use the variable in your tags (i.e. to send the widget ID to Google Analytics) and triggers (i.e. to trigger a tag when a specific widget is used).
+
+## Testing
+
+In order to test Google Tag Manager tracking, you first need to install the widget on your website. Refer to [this guide](../Install%20a%20widget%20to%20your%20website.md) to learn how to do it.
+
+1. Make sure to enable the Google Tag Manager tracking option in the widget settings.
```
Comment (paul-algomo) [inline docs/docs/widgets/Web Analytics/Google Tag Manager.md:49]: ~Make sure to~ Enable the Google Tag Manager....

File: docs/docs/widgets/Web Analytics/Google Tag Manager.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+See the [Variables](#variables) section below to learn how to use event parameters in your triggers.
+
+### Variables
+
+You can use [event parameters](./Overview) as variables in Google Tag Manager:
+
+1. Create a new variable.
+2. Select "Data Layer Variable" as the variable type.
+3. Enter the parameter name under "Data Layer Variable Name", i.e. "widgetId".
```
Comment (paul-algomo) [inline docs/docs/widgets/Web Analytics/Google Tag Manager.md:40]: I think all the `i.e.` occurrences should be `e.g.`

**Review by paul-algomo — APPROVED** [review]
No deal breakers, but I'd suggest changing `i.e.` to `e.g.` and maybe some of the other smaller suggestions

File: docs/docs/widgets/Web Analytics/Google Tag Manager.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+You can use [our events](./Overview) as triggers in Google Tag Manager:
+
+1. Create a new trigger.
+2. Select "Custom Event" as the trigger type.
+3. Enter the event name, i.e. "Message sent (Algomo Widget)".
+4. Save the trigger.
+
+See the [Variables](#variables) section below to learn how to use event parameters in your triggers.
+
+### Variables
```
[the remark this answers is not in the document]
Comment (filipkrw) [reply docs/docs/widgets/Web Analytics/Google Tag Manager.md:34]: No, they might be used by tags, maybe some other entities as well.

**Pull request merged**